docs/dictionary/command/request.lcdoc

```diff
@@ -53,8 +53,7 @@ the eval <Apple event>.
 
 For more information about Apple events, see Apple Computer's technical
 documentation, Inside Macintosh: Interapplication Communication, located
-at <a
-href="http://developer.apple.com/techpubs/mac/IAC/IAC-2.html">http://developer.apple.com/techpubs/mac/IAC/IAC-2.html</a>. 
+at http://developer.apple.com/techpubs/mac/IAC/IAC-2.html
```

Comment:
Putting the link in parenthesis directly after the original in square brackets creates a link, thus:
> \[http://developer.apple.com/techpubs/mac/IAC/IAC-2.html](http://developer.apple.com/techpubs/mac/IAC/IAC-2.html)

Better still, use a simplified version in the square brackets such as:

> located on \[Apple's developer site](http:....)

Additionally, all http references should now be https unless this does not exist based on current best practice.
Further still, the above link does not work anymore! [This page](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/DistrObjects/DistrObjects.html) states that the old way is no longer used. Perhaps this link would be better for now:
https://developer.apple.com/documentation/coreservices/apple_events